tests/test_mocks.py

```diff
@@ -24,7 +24,7 @@
             'hoverProvider': True,
             'completionProvider': {
                 'triggerCharacters': ['.'],
-                'resolveProvider': False
+                'resolveProvider': True
```

Comment:
Is this OK? 
This was the easiest way to test the `resolveProvider` functionality.

Reply:
Sounds good to me.